apps/screloc/main.cpp

```diff
@@ -81,6 +81,8 @@ class Reloc : public Client::Application {
 			                                       "This option should not be mixed with --dump.", &_epFile);
 			commandline().addOption("Input", "replace", "Used in combination with --ep and defines if origins are to be replaced "
 			                                            "by their relocated counterparts or just added to the output.");
+			commandline().addOption("Input", "dropfailure", "Used in combination with --replace/--ep and drops from the output "
```

Comment:
As we are using a dash to separate two words, I prefer `--drop-failure`.

Reply:
sounds good